cfgov/unprocessed/css/molecules/global-search.less

```diff
@@ -102,6 +102,11 @@
         display: block;
     }
 
+    &_input {
+        // Resets for default a-text-input styles.
```

Comment:
this element inherits the following properties from `a-text-input`:
- appearance
- display
- border
- background
- focus state

the only change is to the padding, to make the height match the search button, conforming to the global search pattern design spec.